### PR TITLE
Update to version 1.18

### PR DESCRIPTION
**Pull request by jecnua**
First update of this year. Now the module can access any version up to 1.18.2 (latest).

### CODE REVIEW

File: modules/kubernetes/03-k8s_controllers.tf [hunk 1]
```diff
@@ -71,19 +74,64 @@ resource "aws_elb" "k8s_controllers_internal_elb" {
...
+
   ingress {
     from_port = 6443
     to_port   = 6443
     protocol  = "tcp"
-
-    cidr_blocks = [
-      var.internal_network_cidr,
-    ]
+    cidr_blocks = ["0.0.0.0/0"]
```
Comment (adamwshero) [inline modules/kubernetes/03-k8s_controllers.tf]: you sure you want this wide open? ^^^

File: modules/kubernetes/scripts/bootstrap.tpl [hunk 2]
```diff
@@ -65,12 +63,20 @@ cat << EOF > /etc/apt/sources.list.d/kubernetes.list
...
-apt-get install -y docker.io apt-transport-https awscli jq curl nfs-common
+apt update
+apt install -y \
+  docker.io \
+  apt-transport-https \
+  awscli \
+  jq \
+  curl \
+  nfs-common
 # This need to be synched
```
Comment (adamwday) [inline modules/kubernetes/scripts/bootstrap.tpl:75]: synced

**Review by adamwshero — COMMENTED** [review]
It looks good but I was curious about your ingress rule if that was intentional.

File: modules/kubernetes/03-k8s_controllers.tf (hunk 1 above)
Comment (jecnua) [reply modules/kubernetes/03-k8s_controllers.tf]: You are right. I was testing something and it slipped through. Reverting.

**Comment by jecnua** [comment]
It was not XD reverted. Now merging.

**Pull request merged**